docs/EnterpriseScaleAnalytics-AzureDevOpsDeployment.md

```diff
@@ -77,6 +77,7 @@ To begin, please open the [infra/params.dev.json](/infra/params.dev.json). In th
 | tags | Specifies the tags that you want to apply to all resources. | {`key`: `value`} |
 | administratorPassword | Specifies the administrator password of the sql servers. Will be automatically set in the workflow. **Leave this value as is.** | `<your-secure-password>` |
 | synapseDefaultStorageAccountFileSystemId | Specifies the resource ID of the default storage account file system for synapse. | `/subscriptions/{subscription-id}/resourceGroups/{rg-name}/providers/Microsoft.Storage/storageAccounts/{storage-name}/blobServices/default/containers/{container-name}` |
+| enableDataExplorerPool | Specifies whether an Azure Data Explorer Pool should be deployed inside your Synapse workspace as part of the template. If you selected dataFactory as processingService, leave this value as is. | `true` or `false` |
 | streamanalyticsDefaultStorageAccountFileSystemId | Specifies the resource ID of the default storage account  file system for stream analytics. | `/subscriptions/{subscription-id}/resourceGroups/{rg-name}/providers/Microsoft.Storage/storageAccounts/{storage-name}/blobServices/default/containers/{container-name}` |
 | subnetId | Specifies the resource ID of the subnet to which all services will connect. | `/subscriptions/{subscription-id}/resourceGroups/{rg-name}/providers/Microsoft.Network/virtualNetworks/{vnet-name}/subnets/{subnet-name}` |
 | purviewId | Specifies the resource ID of the central purview instance. | `/subscriptions/{subscription-id}/resourceGroups/{rg-name}/providers/Microsoft.Purview/accounts/{purview-name}` |
```

Comment:
"resource ID" to "Resource ID"
"purview" to "Purview"

Thanks

Reply:
Let's get this done as part of another PR.